activiti-cloud-modeling/templates/deployment.yaml

```diff
@@ -84,11 +84,11 @@ spec:
           value: {{ include "common.keycloak-url" . }}/realms/{{ include "common.keycloak-realm" . }}
         - name: APP_CONFIG_OAUTH2_CLIENTID
           value: {{ include "common.keycloak-client" . | quote }}
-        - name: API_URL
+        - name: APP_CONFIG_BPM_HOST
 {{- if .Values.backend.url }}
           value: {{ .Values.backend.url }}
 {{- else }}
-          value: {{ include "common.gateway-url" $ | quote }}
+          value: {{ include "common.gateway-url" . }}/activiti-cloud-modeling-backend
```

Comment:
@alfresco-build Please, use {{ tpl .Values.backend.prefix . }} instead of hard coding it in the template to be able to override using values if needed.

Reply:
Also, you will need to quote the Yaml value to make it work with Helm install command

Reply:
please take a look